src/Algebra/Apartness/Properties/HeytingField.agda

```diff
@@ -0,0 +1,108 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of Heyting Commutative Rings
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Apartness.Bundles using (HeytingCommutativeRing; HeytingField)
+
+module Algebra.Apartness.Properties.HeytingField
+  {c ℓ₁ ℓ₂} (HF : HeytingField c ℓ₁ ℓ₂) where
+
+open import Function.Base using (_∘_)
+open import Data.Product.Base using (_,_; proj₁; proj₂)
+open import Algebra using (CommutativeRing; RightIdentity; Invertible; LeftInvertible; RightInvertible)
+
+open HeytingField HF
+open CommutativeRing commutativeRing using (ring; *-commutativeMonoid)
+
+open import Algebra.Properties.Ring ring
+  using (-0#≈0#; -‿distribˡ-*; -‿distribʳ-*; -‿anti-homo-+; -‿involutive)
+open import Relation.Binary.Definitions using (Symmetric)
+import Relation.Binary.Reasoning.Setoid as ≈-Reasoning
+open import Algebra.Properties.CommutativeMonoid *-commutativeMonoid
+
+private variable
+  x y z : Carrier
+
+
+open import Algebra.Apartness.Properties.HeytingCommutativeRing heytingCommutativeRing public
```

Comment:
Hmm I was going to say this should go above the variable field by convention, but then I spotted the public at the end. I really feel the `public` key word should have gone in the front of the statement if I could design Agda again...

Reply:
That said, I've now managed to delete this re-export completely, which is one solution to your problem ;-)